src/PVRIptvData.cpp

```diff
@@ -764,14 +764,20 @@ int PVRIptvData::ParseDateTime(std::string& strDate, bool iDateFormat)
 
   std::time_t current_time;
   std::time(&current_time);
-  long offset = std::localtime(&current_time)->tm_gmtoff;
+  long offset = 0;
+#ifndef TARGET_WINDOWS
+  offset = -std::localtime(&current_time)->tm_gmtoff;
+#else
+  _get_timezone(&offset);
+#endif // TARGET_WINDOWS
   
   long offset_of_date = (hours * 60 * 60) + (minutes * 60);
-  if (sign == '-') {
```

Comment:
I didn't change the logic. I've changed sign where it's required (see above at L769 and below L780) because _get_timezone it's actually inverse version of tm_gmtoff.